src/components/Spinner/Spinner.module.scss

```diff
@@ -0,0 +1,24 @@
+.spinner {
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+
+  & svg {
+    display: inherit;
+    -webkit-animation: spin 1s infinite linear;
+    animation: spin 1s infinite linear;
```

Comment:
Are we not using the Autoprefixer PostCSS plugin for palmetto-components?

Reply:
It is not configured yet, but that is the plan!